packages/@aws-cdk-testing/framework-integ/test/aws-ses/test/integ.configuration-set-vdmoptions.ts

```diff
@@ -0,0 +1,40 @@
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as ses from 'aws-cdk-lib/aws-ses';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    new ses.ConfigurationSet(this, 'VdmOptionsEnabled', {
+      vdmOptions: {
+        engagementMetrics: true,
+        optimizedSharedDelivery: true,
+      },
+    });
+
+    new ses.ConfigurationSet(this, 'VdmOptionsDisabled', {
+      vdmOptions: {
+        engagementMetrics: false,
+        optimizedSharedDelivery: false,
+      },
+    });
+
+    new ses.ConfigurationSet(this, 'VdmOptionsNotConfigured', {
+    });
+
+    new ses.VdmAttributes(this, 'VdmAccountLevelSettings', {
+      engagementMetrics: true,
+      optimizedSharedDelivery: true,
+    });
+  }
+}
+
+const app = new App();
+
+new integ.IntegTest(app, 'ConfigurationSetVdmOptionsInteg', {
+  testCases: [new TestStack(app, 'cdk-ses-configuration-set-vdmoptions-integ')],
+});
```

Comment:
It's probably a good idea to run some assertions to make sure that the configuration sets and VdmAttributes are created with the expected values, see [integ-tests](https://docs.aws.amazon.com/cdk/api/v2/docs/integ-tests-alpha-readme.html#assertions). Let me know if you need some guidance

Reply:
@nmussy 
Thank you. I'm not used to it, but I'll give it a try. Please give me a little time.
I will try to implement assertions for account level and configuration set level settings using awsapicall.​​​​​​​​​​​​​​​​

Reply:
@nmussy 
Thank you for your comments. I have added assertions using AwsApiCall for integration testing. Please let me know if there are any inappropriate parts.